.github/workflows/depcheck.yml

```diff
@@ -21,10 +21,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: use node 20.x
+      - name: use node 22.x
         uses: actions/setup-node@v2-beta
         with:
-          node-version: 20.x
+          node-version: 22.x
```

Comment:
this line is missing
${{ runner.os }}-node20.x-yarn-${{ hashFiles('**/yarn.lock') }}

Reply:
Fixed. Thanks!